ethers-core/src/types/transaction/response.rs

```diff
@@ -216,6 +227,17 @@ impl Transaction {
                 rlp.append(&self.input.as_ref());
                 rlp.append(&self.from);
             }
+            // Kroma Deposited Transaction
+            #[cfg(feature = "kroma")]
+            Some(x) if x == U64::from(0x7E) => {
+                rlp_opt(&mut rlp, &self.source_hash);
+                rlp.append(&self.from);
+                rlp_opt(&mut rlp, &self.to);
+                rlp_opt(&mut rlp, &self.mint);
+                rlp.append(&self.value);
+                rlp.append(&self.gas);
+                rlp.append(&self.input.as_ref());
+            }
```

Comment:
Please move it to above the `scroll`, since it's very similar with optimism one.